File: front/src/main/kotlin/com/sns/front/controller/HomeController.java
package com.sns.front.controller;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;

/**
 * @author Hyounglin Jun
 */
@Controller
public class HomeController {
    @RequestMapping("/")
    String indexPage(Model model){
        return "home";
    }
}

File: front/src/main/kotlin/com/sns/front/controller/HomeController.kt
package com.sns.front.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping

/**
 * @author Hyounglin Jun
 */
@Controller
class HomeController {

    @GetMapping("/")
    fun home(): String {
        return "home/home"
    }

}
